### PR TITLE
SimG4 Geometry Unit Tests

### PR DESCRIPTION
**Pull request by ianna**
* Fix volume calculation for Trap
* Use identical tolerance for all tests

### CODE REVIEW

**Comment by ianna** [comment]
please test

**Comment by ianna** [comment]
+1

File: SimG4Core/Geometry/test/Tubs_.cpp [hunk 1]
```diff
@@ -46,10 +46,10 @@ testTubs::matched_g4_and_dd( void )
   std::cout << "\tdd volume = " << ddv << " cm3" <<  std::endl;
   std::cout << "\tDD Information: " << dds << " vol=" << ddsv << " cm3" << std::endl;
 
-  double tolerance = 1e-7;
+  const double tolerance = 1e-7;
   
-  CPPUNIT_ASSERT( fabs( g4v - ddv ) < tolerance );
-  CPPUNIT_ASSERT( fabs( g4v - ddsv ) < tolerance );
+  CPPUNIT_ASSERT( abs( g4v - ddv ) < tolerance );
```
Comment (civanch) [inline SimG4Core/Geometry/test/Tubs_.cpp]: may be std::abs ?

**Comment by fabiocos** [comment]
@civanch @ianna can we take the opportunity to add std::abs also in the Cons test that was merged?

**Comment by civanch** [comment]
please test

**Comment by ianna** [comment]
+1

**Comment by civanch** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**